keyboards/handwired/mo_keypad_24/keymaps/default/keymap.c

```diff
@@ -0,0 +1,381 @@
+/* Copyright 2021 mo
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+typedef enum _layer_names {
+    _BASE,
+    _FN,
+    // add new ones above here
+    _JUMP,
+    _LAYER_COUNT
+} layer_names;
+
+typedef enum dancy_ {
```

Comment:
Could this be simplified?  Eg, generally we want the default keymap to be as plain and simple as possible.  Namely, because the new keymap script copies the default, and we want to avoid having a lot of complexity for newer users. 

However, having another "stock" keymap with more complexity is fine.